src/components/button-toggle/button-toggle.style.ts

```diff
@@ -40,7 +41,18 @@ const StyledButtonToggleContentWrapper = styled.div`
   height: 100%;
 `;
 
-const StyledButtonToggleLabel = styled.label`
+export interface StyledButtonToggleLabelProps {
+  /** buttonIcon to render. */
+  buttonIcon?: IconType;
+  /** Sets the size of the buttonIcon (eg. large) */
+  buttonIconSize?: ButtonToggleIconSizes;
+  /** Disable all user interaction. */
+  disabled?: boolean;
+  /** ButtonToggle size */
+  size: "small" | "medium" | "large";
```

Comment:
```suggestion
  /** ButtonToggle size */
  size?: "small" | "medium" | "large";
```

Since consumers may not be specifying the `size` prop (even though we set its default value ourselves in `button-toggle.component.tsx`) the prop's type should be optional to prevent consumers receiving an error.

Reply:
this is an internal component, the default size is `medium` and it's being set here:
https://github.com/Sage/carbon/pull/5365/files#diff-3f6aa630f9b196e239402e9dd6a2a350de858f8f653eb9e092430a971d60c695R52